projects/memapCore/src/main/java/memap/components/prototypes/Consumer.java

```diff
@@ -29,11 +29,11 @@ public Consumer(String name, int port) {
 		super(name, port);
 	}
 
-	public DemandMessage consumptionMessage = new DemandMessage();
```

Comment:
That makes sense 👍

Reply:
yes, but it might interfere with the GUI code. Has to be checked @daniagudelos

Reply:
the simulation runs through - there shouldn't be issues.